apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -426,156 +431,173 @@ class ManageStudentsTable extends Component {
 
   getColumns(sortable) {
     const {loginType} = this.props;
-    let dataColumns = [
```

Comment:
I think you'll see this pattern all over LP land. Would be nice to sneak in a refactor of other tables as an incremental follow-up.